### PR TITLE
#fix 3687: shells

### PR DESCRIPTION
**Pull request by dualfade**
please review.

Thank you.

### CODE REVIEW

File: packages/shells/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=shells
```
[the remark this answers is not in the document]
Comment (dualfade) [reply packages/shells/PKGBUILD:4]: Will check with 4nd34z. Maybe alias it ?? Thanks !

[the remark this answers is not in the document]
Comment (dualfade) [reply packages/shells/PKGBUILD:4]: Alias of "shellz" ? I do not see that in aur either. Let me know your thoughts. 

Thanks.

[the remark this answers is not in the document]
Comment (noraj) [reply packages/shells/PKGBUILD:4]: Or maybe shells-4nd34z, like we do when there are duplicate names.

image

[the remark this answers is not in the document]
Comment (dualfade) [reply packages/shells/PKGBUILD:4]: shellz would be ideal.  If we're ok with that, I will update the PKGBUILD and re push.

Appreciate it.

[the remark this answers is not in the document]
Comment (dualfade) [reply packages/shells/PKGBUILD:4]: Are we good on this or is there something else you guys want to do ?

Lmk. Thanks --

[the remark this answers is not in the document]
Comment (noraj) [reply packages/shells/PKGBUILD:4]: Actually I'm not sure. pkgname is still shells, only _pkgname was changed. I would have liked noptrix review on this one as I'm not sure what the aproach to adopt here.

[the remark this answers is not in the document]
Comment (dualfade) [reply packages/shells/PKGBUILD:4]: Ok lmk me know. That is the repo name. Not sure then either is that is an issue. Thanks.

[the remark this answers is not in the document]
Comment (Prijey) [reply packages/shells/PKGBUILD:4]: Hello, I have such a mistake, I would appreciate if you can help. Thanks
error filed retrieving file 'community.db
error filed retrieving file  (download library error)

**Comment by dualfade** [comment]
What does this ^^ have to do with "shells" ???

**Pull request merged**